Stack/Opc.Ua.Core/Opc.Ua.Core.csproj

```diff
@@ -36,54 +36,17 @@
     <PackageReference Include="Newtonsoft.Json" Version="10.0.3" />
   </ItemGroup>
 
-  <ItemGroup Condition="'$(TargetFramework)' == 'netstandard1.3'">
-    <PackageReference Include="System.Data.Common" Version="4.3.0" />
-    <PackageReference Include="System.ServiceModel.Primitives" Version="4.5.3" />
-    <PackageReference Include="System.Net.NameResolution" Version="4.3.0" />
-    <PackageReference Include="System.Xml.XmlSerializer" Version="4.3.0" />
-    <PackageReference Include="System.Collections.NonGeneric" Version="4.3.0" />
-    <PackageReference Include="System.Reflection.TypeExtensions" Version="4.4.0" />
-    <PackageReference Include="System.Net.Websockets.Client" Version="4.3.2" />
-  </ItemGroup>
-  <ItemGroup Condition="'$(TargetFramework)' == 'netstandard1.3' AND $(DefineConstants.Contains('NO_HTTPS'))=='false'">
-    <PackageReference Include="Microsoft.AspNetCore.Server.Kestrel" Version="1.1.3" />
-    <PackageReference Include="Microsoft.AspNetCore.Server.Kestrel.Https" Version="1.1.3" />
-  </ItemGroup>
-
-  <ItemGroup Condition="'$(TargetFramework)' == 'netstandard1.4'">
-    <PackageReference Include="System.Data.Common" Version="4.3.0" />
-    <PackageReference Include="System.ServiceModel.Primitives" Version="4.5.3" />
-    <PackageReference Include="System.Net.NameResolution" Version="4.3.0" />
-    <PackageReference Include="System.Xml.XmlSerializer" Version="4.3.0" />
-    <PackageReference Include="System.Collections.NonGeneric" Version="4.3.0" />
-    <PackageReference Include="System.Reflection.TypeExtensions" Version="4.4.0" />
-    <PackageReference Include="System.Net.Websockets.Client" Version="4.3.2" />
-  </ItemGroup>
-  <ItemGroup Condition="'$(TargetFramework)' == 'netstandard1.4' AND $(DefineConstants.Contains('NO_HTTPS'))=='false'">
-    <PackageReference Include="Microsoft.AspNetCore.Server.Kestrel" Version="1.1.3" />
-    <PackageReference Include="Microsoft.AspNetCore.Server.Kestrel.Https" Version="1.1.3" />
-  </ItemGroup>
-
-  <ItemGroup Condition="'$(TargetFramework)' == 'net46'">
-    <Reference Include="System.IdentityModel" />
-    <Reference Include="System.IdentityModel.Selectors" />
-    <Reference Include="System.Security" />
-    <Reference Include="System.ServiceModel" />
-  </ItemGroup>
-  <ItemGroup Condition="'$(TargetFramework)' == 'net46' AND $(DefineConstants.Contains('NO_HTTPS'))=='false'">
-    <PackageReference Include="Microsoft.AspNetCore.Server.Kestrel" Version="1.1.3" />
-    <PackageReference Include="Microsoft.AspNetCore.Server.Kestrel.Https" Version="1.1.3" />
-  </ItemGroup>
-
-  <ItemGroup Condition="'$(TargetFramework)' == 'net451'">
+  <ItemGroup Condition="'$(TargetFramework)' == 'net462'">
     <Reference Include="System.IdentityModel" />
     <Reference Include="System.IdentityModel.Selectors" />
     <Reference Include="System.Security" />
     <Reference Include="System.ServiceModel" />
+    <Reference Include="System.Net.Http" Version="4.3.0" />
```

Comment:
><Reference Include="System.Net.Http" Version="4.3.0" /> [](start = 4, length = 55)

does this cause a new warning in .Net projects? Or shouldn't this ref have no version?

Reply:
You are right, it was a leftover to force deal with bug in old mono build (for travis ci), I've removed it that the default version from targetframework will be used.